imports/collections/users_private.js

```diff
@@ -18,6 +18,29 @@ Meteor.methods({
         console.log(user);
     },
 
+    'users.editProfile'(userId, eMail, longName) {
+        if (Meteor.isServer) {
```

Comment:
I think the `Meteor.isServer`-condition is not necessary. Following the meteor style theses special meteor methods run on both the client and the server. Executing them on the client gives us the optimistic ui which leads to fast reactions visible for the user. Executing them on the server makes sure that the security checks are secure ;-).

Wrapping the whole code inside a `if (Meteor.isServer)`-branch leads to loosing the optimistic ui benefit. The client executes the method and is done immediately without doing anything at all and the ui will be updated after the server is done.